### PR TITLE
Fix #2146: [RTL] Highfi CompletedStoryList #2146

### LINKED ISSUE #2146: [RTL] Highfi CompletedStoryList
`CompletedStoryList` needs RTL support as shown in below screen.
UP (Stories Completed)

Make sure that this works correctly in mobile (landscape + portrait) and tablet (landscape + portrait).

## References
Mobile Portrait: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/grid/
Mobile Landscape: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/grid/
Tablet: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/grid/
Wiki: https://github.com/oppia/oppia-android/wiki/Working-on-UI

### PR DESCRIPTION
**Pull request by FareesHussain**
## Explanation

Fixes #2146 

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

Mobile Portrait | Mobile Landscape 
--- | --- 
 | 

Tablet Portrait | Tablet Landscape 
--- | --- 
 |

### CODE REVIEW

**Comment by FareesHussain** [comment]
@rt4914 please check

This is what I've done so far as the only changes needed for RTL were in TextView remaining were automatically managed

**Comment by anandwana001** [comment]
@FareesHussain It looks like tests in `CompletedStoryListActivityTest ` are failing, as this pr involves modifying landscape layout files.

**Comment by rt4914** [comment]
> @FareesHussain It looks like tests in `CompletedStoryListActivityTest ` are failing, as this pr involves modifying landscape layout files.

Actually thats an ongoing issue https://github.com/oppia/oppia-android/issues/1658 .
For now we will have to run this and check.

File: app/src/main/res/layout/completed_story_item.xml [hunk 1]
```diff
@@ -21,7 +21,6 @@
     android:layout_marginBottom="12dp"
     android:clickable="true"
     android:focusable="true"
-    android:onClick="@{(v) -> viewModel.onCompletedStoryItemClicked()}"
```
Comment (rt4914) [inline app/src/main/res/layout/completed_story_item.xml]: Please add this click item back.

**Review by rt4914 — CHANGES_REQUESTED** [review]
This PR looks good but one change is suggested.

Also I am adding screenshots that I recorded during testing for future reference.
Screenshot_1606333046
Screenshot_1606333063
Screenshot_1606333107
Screenshot_1606333118

File: app/src/main/res/layout/completed_story_item.xml (hunk 1 above)
Comment (FareesHussain) [reply app/src/main/res/layout/completed_story_item.xml]: Done

**Comment by FareesHussain** [comment]
May be the previous checks failed cause I removed the onClick attribute

**Comment by rt4914** [comment]
> May be the previous checks failed cause I removed the onClick attribute

Yes, exactly.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**